integration/react/package.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
as flat bundling will be a breaking change (no more reaching out to internal directory structure - which is even proposed in the main READMEs on several occasions) I'd go with simple `redux-persist/react` proxy module, I've attempted to keep both `redux-persist/integration/react` and `redux-persist/react` here but there is really no point in that

Reply:
I agree, `redux-persist/react` is probably best. No issues with breaking changes, although I do have a couple of breaking changes in mind, so we can stack these together for a v6